quasar.conf.js

```diff
@@ -34,7 +34,6 @@ module.exports = function() {
     // Full list of options: https://quasar.dev/quasar-cli/quasar-conf-js#Property%3A-build
     build: {
       vueRouterMode: "hash", // available values: 'hash', 'history'
-      env: require("./src/boot/env-parser")(),
```

Comment:
we can also remove env from the dependencies.. and also from the boot folder

Reply:
OK, I removed the `dotenv` dependency and the `env-parser.js`